public/js/term-highlight.js

```diff
@@ -0,0 +1,34 @@
+jQuery(function($) {
```

Comment:
@beseven Ideally yes. However, there are a few things that would sway me to conclude it is ok to go in, at least until the next release:
* First off, I'd like to understand why the JS implementation was done as the default
* A JS solution is going to be required when type-ahead is implemented
* As @neilbmclaughlin says ES has an option to return the highlighting as part of the results
* Very little traffic is going to the site atm, especially with the banners up on Choices
I'm really in two minds about this...